src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -15,9 +16,11 @@ public class MovieSession {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "movie_id")
     private Movie movie;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```
Many movie sessions can be at the same cinema hall (not at the same time)